ticdc/troubleshoot-ticdc.md

```diff
@@ -61,7 +61,8 @@ cdc cli changefeed create --pd=http://10.0.10.25:2379 --start-ts=415238226621235
 
 > **Note:**
 >
-> TiCDC (the 4.0 version) does not support dynamically modifying the file sorting and memory sorting yet.
+> + TiCDC (the 4.0 version) does not support dynamically modifying the file sorting and memory sorting yet.
+> + Currently, the file sorting feature only has limited processing capacity. If too large a  data volume on a single table causes the file sorting to fail, you can modify the task configuration of TiCDC to filter this table and use other backup and restore tools such as [BR](/br/backup-and-restore-tool.md) to restore the table before you resume replicating the table.
```

Comment:
```suggestion
> + Currently, the file sorting feature only has limited processing capacity. If too large a data volume on a single table causes the file sorting to fail, you can modify the task configuration of TiCDC to filter out this table and use other backup and restore tools (such as [BR](/br/backup-and-restore-tool.md)) to restore the table before you resume replicating the table.
```

Reply:
Applied in https://github.com/pingcap/docs/pull/3881#discussion_r486778489.